pyproject.toml

```diff
@@ -77,6 +77,7 @@ addopts = ["--cov=snapcraft"]
 [tool.ruff]
 target-version = "py310"
 line-length = 88
+target-version = "py310"
```

Comment:
Isn't this already here 2 lines above?

Reply:
Strangely enough, it looks like this is causing the errors in CI of `KeyError: 'lint-docs'`

Reply:
This is what I get for rebasing without doing a thorough check of the results!

```suggestion
```